From 672961af6ace0285a3f02598a2c8112f887e85e3 Mon Sep 17 00:00:00 2001
From: Changqing Li <changqing.li@windriver.com>
Date: Tue, 13 Aug 2019 10:44:42 +0800
Subject: [PATCH] vrf-helper: fix cmd for get DISTRIB_RELEASE

DISTRIB_RELEASE in lsb-release is in format like:
DISTRIB_RELEASE=10.19.32.0

we need to get 10.19.32.0

Upstream-Status: Inappropriate [WRL specific]

Signed-off-by: Changqing Li <changqing.li@windriver.com>
---
 src/vrf-helper.in | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/vrf-helper.in b/src/vrf-helper.in
index 960cc75..ec88b96 100755
--- a/src/vrf-helper.in
+++ b/src/vrf-helper.in
@@ -375,8 +375,7 @@ initialize_env () {
                                   cut -d " " -f4)
     elif [ -e /etc/lsb-release ]; then
         VRF_DISTRIB_RELEASE_VER=$(grep "DISTRIB_RELEASE" /etc/lsb-release | \
-                                  cut -d "=" -f2 | cut -d "+" -f1 |
-                                  cut -d '.' -f1-2 )
+                                  cut -d "=" -f2)
         VRF_DISTRIB_ID=$(grep "DISTRIB_ID=" /etc/lsb-release| \
                                   sed -e "s/DISTRIB_ID=//")
     elif [ -e /etc/os-release ]; then
-- 
2.7.4


